ctlog/ctlog.go

```diff
@@ -0,0 +1,57 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package ctlog contains data structures and methods to do with CT Log metadata
+// that is needed by the monitor.
+//
+// TODO(katjoyce): Try to come up with a better package name.
+package ctlog
+
+import (
+	"crypto"
+	"fmt"
+
+	ct "github.com/google/certificate-transparency-go"
+	"github.com/google/certificate-transparency-go/logid"
+)
+
+// Log represents a CT Log and contains the Log metadata needed by the monitor.
```

Comment:
"represents a CT log" is a bit vague if this is the main type of the package. Perhaps add some indication of where and how it's used as well as what it contains.

Reply:
So, I've tried a bunch, and I'm not sure how exactly to describe where and how it's going to be used at this point... it'll be used all over the place, in basically every piece of the monitor that gets added, for a variety of different things... I've removed the vague bit of language though?

Reply:
OK. It seems like it's an important type. Can evolve the comment later.